### ISSUE TITLE
Getting a -Infinity when specifying 2 constraints instead of 1

### ISSUE THREAD

**Issue by elizeng** [mmatschiner/tutorials#20]
I have been able to successfully follow the tutorial to get a divergence time estimation when I specified one node for calibration. But as soon as I specify 2 it gives me -Infinity.

The following is the constraint that I have specified in my 2 constraint run with the first one having worked before on its own.
lognormal(0,2.95,0.30)	crown	SF,NF
lognormal(0,4.56,0.26)	crown	SF,NF,SGP,NGP

The species involved are SF, NF, SGP, NGP, APT, SP, CP. 
The following are monophyletic clades:
SF,NF
SGP,NGP
SF,NF,SGP,NGP

Is it not possible to have 2 point of calibration within a larger clade or does something need to be modified.

xml file is as attached (as a .txt file)
[input.txt](https://github.com/mmatschiner/tutorials/files/11013324/input.txt)

**Comment by mmatschiner** [comment]
Hi, have a look here: https://www.beast2.org/2018/07/04/fatal-errors.htmlAdding a starting tree might help.On 20 Mar 2023, at 04:09, Elize Ng ***@***.***> wrote:﻿
I have been able to successfully follow the tutorial to get a divergence time estimation when I specified one node for calibration. But as soon as I specify 2 it gives me -Infinity.
The following is the constraint that I have specified in my 2 constraint run with the first one having worked before on its own.
lognormal(0,2.95,0.30)	crown	SF,NF
lognormal(0,4.56,0.26)	crown	SF,NF,SGP,NGP
The species involved are SF, NF, SGP, NGP, APT, SP, CP.
The following are monophyletic clades:
SF,NF
SGP,NGP
SF,NF,SGP,NGP
Is it not possible to have 2 point of calibration within a larger clade or does something need to be modified.
xml file is as attached (as a .txt file)
input.txt
